chart/openfaas/templates/queueworker-dep.yaml

```diff
@@ -21,11 +21,13 @@ spec:
       - name:  queue-worker
         image: {{ .Values.queueWorker.image }}
         imagePullPolicy: {{ .Values.openfaasImagePullPolicy }}
-        {{- if .Values.functionNamespace }}
         env:
+        {{- if .Values.functionNamespace }}
         - name: faas_function_suffix
           value: .{{ .Values.functionNamespace }}
         - name: ack_wait    # Max duration of any async task / request
```

Comment:
Hi Eric,

The `end` for this statement should finish after the first env-var, not after ack_wait.

Please can you fix?

Alex